gradle/publish.gradle

```diff
@@ -123,7 +123,12 @@ configure(scalaProjects) { project ->
 
     task scaladocJar(type: Jar) {
         from scaladoc
-        classifier = 'scaladoc'
+        classifier = 'javadoc'
+
+        // Workaround for gradle 2.13 scaladoc issue: https://github.com/gradle/gradle/issues/9855
+        if (project.scalaVersion.startsWith("2.13")) {
+            project.scaladoc.scalaClasspath += files("$rootDir/gradle/scala/lib/scala-ant-2.13.1.jar")
```

Comment:
How does this file get to this location?

Reply:
I compiled it and added it to this location.

Reply:
Source code for the jar: https://github.com/cigaly/scala-ant

Reply:
Oh, I totally missed the new file in the PR because it's binary